.github/workflows/manual.yml

```diff
@@ -37,13 +37,10 @@ jobs:
          uses: azure/CLI@v1
          id: fetch-secrets
          with:
-           inlineScript: |
-             SECRET_VALUE=$(az keyvault secret show --name "SLACK-WEBHOOK" --vault-name "${{ secrets.INFRA_KEY_VAULT}}" --query "value" -o tsv)
-             echo "::add-mask::$SECRET_VALUE"
-             echo "SLACK-WEBHOOK=$SECRET_VALUE" >> $GITHUB_OUTPUT
-             SECRET_VALUE=$(az keyvault secret show --name "SLACK-RELEASE-NOTE-WEBHOOK" --vault-name "${{ secrets.INFRA_KEY_VAULT}}" --query "value" -o tsv)
-             echo "::add-mask::$SECRET_VALUE"
-             echo "SLACK-RELEASE-NOTE-WEBHOOK=$SECRET_VALUE" >> $GITHUB_OUTPUT
+          inlineScript: |
+            SECRET_VALUE=$(az keyvault secret show --name "SLACK-WEBHOOK" --vault-name "${{ secrets.INFRA_KEY_VAULT}}" --query "value" -o tsv)
+            echo "::add-mask::$SECRET_VALUE"
+            echo "SLACK-WEBHOOK=$SECRET_VALUE" >> $GITHUB_OUTPUT
 
```

Comment:
seems reference to slack webhook is wrong below

https://github.com/DFE-Digital/get-into-teaching-api/blob/6db1048f3e7b25da8db667048ec5c3f456a2dd45/.github/workflows/manual.yml#L78C66-L78C92